packages/commonwealth/client/scripts/state/api/threads/getThreadsById.ts

```diff
@@ -0,0 +1,40 @@
+import { useQuery } from '@tanstack/react-query';
+import axios from 'axios';
+import Thread from 'models/Thread';
+import app from 'state';
+import { ApiEndpoints } from 'state/api/config';
+
+const THREAD_STALE_TIME = 30 * 1_000; // 30 s
+
+interface GetThreadsByIdProps {
+  chainId: string;
+  ids: number[];
+}
+
+const getThreadsById = async ({ chainId, ids }: GetThreadsByIdProps) => {
+  const response = await axios.get(
+    `${app.serverUrl()}${ApiEndpoints.FETCH_THREADS}`,
+    {
+      params: {
+        chain: chainId,
+        thread_ids: ids
+      },
+    }
+  );
+
+  // TODO: do we need to do this?
+  // app.chainEntities.getRawEntities(app.activeChainId()),
```

Comment:
@jnaviask to confirm if we still need this, slack discussion [here](https://commonxyz.slack.com/archives/C050AE0URFH/p1690579662596919)

Reply:
if it's already commented out, probs safe to omit it @mzparacha

Reply:
Oh i think we had a discussion in [slack](https://commonxyz.slack.com/archives/C050AE0URFH/p1690825019167619) to keep it? -- the comment I left was older